docs/source/api/v3/servercheck.rst

```diff
@@ -31,7 +31,34 @@ Fetches identifying and meta information as well as "check" values regarding all
 
 Request Structure
 -----------------
-No parameters available.
+.. table:: Request Query Parameters
+
+	+-----------+------------------------------------------------------------------------------------+
+	| Name      | Description                                                                        |
+	+===========+====================================================================================+
+	| id        | Return only :term:`Servers` with this integral, unique identifier (id)             |
+	+-----------+------------------------------------------------------------------------------------+
```

Comment:
`Servers` isn't a `term`. You can add it to the glossary if you like, but I don't think I would bother. I'd probably just not use `term`.

Reply:
Removed term

Reply:
[*Cache Server*](https://traffic-control-cdn.readthedocs.io/en/latest/glossary.html#term-Cache-Server) is a term and IMO would be appropriate here.